storage/innobase/handler/ha_innodb.cc

```diff
@@ -10948,15 +10948,19 @@ create_table_info_t::create_table_def()
 
 	heap = mem_heap_create(1000);
 
+    bool have_vers_start = false, have_vers_end = false;
+
 	for (ulint i = 0; i < n_cols; i++) {
 		Field*	field = m_form->field[i];
 		ulint vers_row = 0;
 
 		if (m_form->versioned()) {
 			if (i == m_form->s->row_start_field) {
 				vers_row = DATA_VERS_START;
+				have_vers_start = true;
 			} else if (i == m_form->s->row_end_field) {
 				vers_row = DATA_VERS_END;
+				have_vers_end = true;
```

Comment:
Use `ut_d()` also for these added lines.

Reply:
:+1: